test/utils/dynamicTypings.js

```diff
@@ -0,0 +1,35 @@
+const _ = require("lodash");
+
+const {
+  getWidgetTypeDeclarations,
+  getPageElementsTypeDeclarations,
+  getAmbientModuleDeclaration,
+  getJswTypeDeclaration
+} = require("../../dist/corvidTypes.umd");
+
+module.exports = function getDynamicTypings({
```

Comment:
why do we need this redundant `function getDynamicTypings`?

Reply:
Cause we had the exact same code between positive and negative checks, and it was getting more complex. so instead of changing both places I extracted this function